src/lib/tvgMath.h

```diff
@@ -43,7 +43,7 @@ static inline bool mathEqual(float a, float b)
 
 static inline bool mathRightAngle(const Matrix* m)
 {
-   auto radian = fabsf(atan2(m->e21, m->e11));
+   auto radian = fabsf((float)atan2(m->e21, m->e11));
```

Comment:
Can we use atan2f() ??

Reply:
I had to be very tired writing this... :D thx